src/assets/i18n/en.json

```
@@ -1,11 +1,35 @@
 {
-  "HOME": {
-    "TITLE": "Hello Angular with ngx-translate!",
-    "SELECT": "Change language"
+  "shows": {
+    "movie-list": {
+      "TITLE": "Movies",
+      "LABEL": {
```

```suggestion
      "labels": {
```
hoa thì hoa hết e, còn thường thì thường hết, đừng để lẫn lộn như này 😅
còn a thì hay để thường hết, chữ hoa là cho enum value